setup.py

```diff
@@ -1,25 +1,26 @@
-from setuptools import setup
+import setuptools
 
 with open('README.md', 'r') as fh:
     long_description = fh.read()
 
-setup(name='SalesforcePy',
-      version='1.0.1',
-      description='An absurdly simple package for making Salesforce Rest API requests',
-      url='https://github.com/forcedotcom/SalesforcePy',
-      author='Aaron Caffrey',
-      author_email='acaffrey@salesforce.com',
-      license='BSD',
-      long_description=long_description,
-      long_description_content_type='text/markdown',
-      classifiers=[
-          'Programming Language :: Python :: 3',
-          'License :: OSI Approved :: BSD License',
-          'Operating System :: OS Independent',
-      ],
-      packages=['SalesforcePy'],
-      zip_safe=False,
-      package_dir={'SalesforcePy': 'src'},
-      install_requires=['requests', 'pytest',
-                        'responses', 'coverage', 'python-coveralls'],
-      tests_require=['pytest', 'responses'])
+setuptools.setup(
+    name='SalesforcePy',
+    version='1.0.2',
+    description='An absurdly simple package for making Salesforce Rest API requests',
+    url='https://github.com/forcedotcom/SalesforcePy',
+    author='Aaron Caffrey',
+    author_email='acaffrey@salesforce.com',
+    license='BSD',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    classifiers=[
+      'Programming Language :: Python :: 3',
+      'License :: OSI Approved :: BSD License',
+      'Operating System :: OS Independent',
+    ],
+    packages=setuptools.find_packages(),
+    zip_safe=False,
+    install_requires=['requests', 'pytest',
+                    'responses', 'coverage'],
```

Comment:
Forgot about removing python-coveralls here, will fix that. Problem was that coverage needs to be pinned as there's a dependency conflict between unpinned coverage and the version that python-coveralls depends on.